test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,69 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import org.junit.Test;
+
+import seedu.addressbook.common.Utils;
+
+/**
+ * Unit tests for Utils class
+ * 
+ * @author Vivian
+ */
+public class UtilsTest {
+
+    /**
+     * Test method for {@link seedu.addressbook.common.Utils#isAnyNull(java.lang.Object[])}.
+     */
+    @Test
+    public void testIsAnyNull_noNullArgsReturnFalse() {
```

Comment:
you have followed the convention for test names, albeit minor error, separate the three parts with an underscore for better readability